benchmark/CMakeLists.txt

```diff
@@ -29,7 +29,7 @@ set_target_properties(quicr_benchmark
         CXX_STANDARD_REQUIRED YES
         CXX_EXTENSIONS ON)
 
-if(CLANG_TIDY_EXE)
+if(LINT)
     set_target_properties(quicr_benchmark PROPERTIES CXX_CLANG_TIDY "${CLANG_TIDY_EXE}")
 endif()
```

Comment:
Might be good to keep the original check as well, in case clang-tidy is not installed?

Reply:
Yeah I had debated this actually. I did change it to REQUIRED so in theory it MUST exist but think on review I'll actually add a cmake function to set this on our targets for consistency and can check it exists there.